omero-installation/omero_installation_debian8_ice35.md

```diff
@@ -0,0 +1,157 @@
+### OMERO.server installation on Debian 8 and Ice 3.5
+
+This is an example walkthrough for installing OMERO 5.2.x on Debian 8, using a dedicated system use.
+These instructions assume your Linux distribution is configured with a UTF-8 locale.
+
+#### Name: OMERO team
+
+#### Institute: University of Dundee
+
+#### Contact: OME External Developer List <ome-devel@lists.openmicroscopy.org.uk>
+
+#### Context:
+
+For convenience in this walkthrough the main OMERO configuration options have been defined as environment variables. When following this walkthrough you can either use your own values:
+ 
+  	OMERO_DB_USER=db_user
+	OMERO_DB_PASS=db_password
+	OMERO_DB_NAME=omero_database
+	OMERO_ROOT_PASS=omero_root_password
+	OMERO_DATA_DIR=/OMERO
+
+	OMERO_WEB_PORT=80
+
+	export OMERO_DB_USER OMERO_DB_PASS OMERO_DB_NAME OMERO_ROOT_PASS OMERO_DATA_DIR OMERO_WEB_PORT
+
+	export PGPASSWORD="$OMERO_DB_PASS"
+
+#### System Specifications:
+
+Debian 8 with Java 1.8, Ice 3.5 and PostgreSQL 9.4. Nginx 1.8 for OMERO.web
+
+#### Installation:
+
+The following steps describe how to install an OMERO.server and to deploy OMERO.web
+using Nginx.
+
+##### Step 1:
+
+As **root**, Install:
+
+Java 1.8:
+
+	echo 'deb http://httpredir.debian.org/debian jessie-backports main' > /etc/apt/sources.list.d/jessie-backports.list
+	apt-get update
+	apt-get -y install openjdk-8-jre-headless
+
+Various dependencies:
+	
+	apt-get -y install \
+	python-{matplotlib,numpy,pip,scipy,tables,virtualenv}
+
+	apt-get -y install \
+	libtiff5-dev \
+	libjpeg62-turbo-dev \
+	zlib1g-dev \
+	libfreetype6-dev \
+	liblcms2-dev \
+	libwebp-dev \
+	tcl8.6-dev \
+	tk8.6-dev
+
+	pip install --upgrade pip
+
+Pillow: 
+	
+	pip install Pillow<3.0
+
+Ice 3.5:
+
+	apt-get -y install ice-services python-zeroc-ice
+
+Postgres 9.4:
+
+	apt-get -y install postgresql
+
+##### Step 2:
+
+As **root**, create an omero system user and directory for the OMERO repository
+
+	useradd -m omero
+	chmod a+X ~omero
+
+	mkdir -p "$OMERO_DATA_DIR"
+	chown omero "$OMERO_DATA_DIR"
+
+##### Step 3:
+
+As **root**, create a database user and a database
+
+	echo "CREATE USER $OMERO_DB_USER PASSWORD '$OMERO_DB_PASS'" | \
+    su - postgres -c psql
+	su - postgres -c "createdb -E UTF8 -O '$OMERO_DB_USER' '$OMERO_DB_NAME'"
+
+	psql -P pager=off -h localhost -U "$OMERO_DB_USER" -l
+
+##### Step 4:
+
+As the **omero** system user,
+
+Install the latest 5.2 OMERO.server:
+
+	cd ~omero
+	SERVER=http://downloads.openmicroscopy.org/latest/omero5.2/server-ice35.zip
+	wget $SERVER -O OMERO.server-ice35.zip
+	unzip -q OMERO.server*
```

Comment:
I took that part out of the doc since I assume that was already installed. 
I will add note at the top same for ``wget``